### PR TITLE
Implement custom colors to default theme

### PR DESCRIPTION
**Pull request by dima-diachkov**
1. added selected custom colors
2. added ThemeProvider to App

### CODE REVIEW

**Review by sergeyman — DISMISSED** [review]

File: src/customizing/customTheme.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  palette: {
+    primary: {
+      main: "#5fafbe",
+      light: "#92e1f1",
+      dark: "#297f8e"
+    },
+    secondary: {
+      main: "#f8d8cc",
+      light: "#ffffff",
+      dark: "#c5a79b"
```
Comment (kerrimov) [inline src/customizing/customTheme.tsx:14]: please move these colors to `shared/variables.scss`

Comment (dima-diachkov) [reply src/customizing/customTheme.tsx:14]: colors have been moved to shared/variables.scss.

**Review by maria-khudolii — DISMISSED** [review]
ok

File: src/shared/variables.scss.d.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export interface IVariables {
+    primaryMain: string;
+    primaryLight: string;
+    primaryDark: string;
+    secondaryMain: string;
+    secondaryLight: string;
+    secondaryDark: string;
+}
```
Comment (kerrimov) [inline src/shared/variables.scss.d.ts:8]: Good decision. Please remove `I` from the interface

**Review by kerrimov — APPROVED** [review]

**Pull request merged**